### ISSUE TITLE
Add Method to Enable/Disable Camera

### ISSUE THREAD

**Issue by jimboca** [camect/camect-py#7]

**Comment by chao-camect** [comment]
You can use [enable_alert / disable_alert](https://github.com/camect/camect-py/blob/master/camect/__init__.py#L164).

**Comment by jimboca** [comment]
Yes of course I am aware and use that. But if the camera is disabled in Home mode I can't re-anable it from my home automation.

**Comment by chao-camect** [comment]
The home mode has priority. We don't have a way enable cameras that's disabled by home mode.
Can you take me a bit about the use case here?

**Comment by jimboca** [comment]
Sometimes when I am Home, I just want to enable one camera that is typically off when I am home, and since my Home Automation App has all cameras listed that have ever been enabled on that hub it would be easiest just to enable it thru that App, or with my home automation routines.

I can do this in the PWA by showing all disabled cameras, then enabling that one, so I was hoping the same functionality could be exposed in API.
